external/model-preparation-algorithm/mpa_tasks/apis/task.py

```diff
@@ -181,6 +184,30 @@ def _initialize(self, dataset=None, output_model=None, export=False):
                 del self._model_cfg['fp16']
         self._precision = [ModelPrecision.FP32]
 
+        # Add/remove adaptive interval hook
+        if self._recipe_cfg.get('use_adaptive_interval', False):
+            self._recipe_cfg.adaptive_validation_interval = self._recipe_cfg.get('adaptive_validation_interval', dict(max_interval=5))
```

Comment:
`self._recipe_cfg` is `ConfigDict`. Therefore, `self._recipe_cfg.adaptive_validation_interval` will raise the `AttributeError` if the key `adaptive_validation_interval` does not exist.

Reply:
Sorry, I don't understand. As far as I know, ConfigDict can insert keys and values directly by using `self._recipe_cfg.adaptive_validation_interval`. I tested it works.

Reply:
I guess I got it wrong. I'm sorry.